FileFormats_details.md

```diff
@@ -0,0 +1,22 @@
+## File Formats
+
+ * HDF5/NetCDF, GRIB, UM files: 
+     * Overview of format
+     * ncdump
+     * ncview
+     * GRADs  (do we want it in?, if not, what for GRIB?)
```

Comment:
If someone knows how to use it. otherwise there's eccodes and I think an Iris extension for grib files